development/mimir-microservices-mode/config/mimir.yaml

```diff
@@ -18,8 +18,6 @@ ingester_client:
 
 ingester:
   ring:
-    # We want to start immediately.
```

Comment:
am I misunderstanding what final_sleep does or does this comment not make sense?

Reply:
I think the comment was just wrong. Maybe it used to apply to `join_after`?